src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/blocks/emptyblock/InputEmptyBlockCaseAndDefaultWithTextOption.java

```diff
@@ -0,0 +1,94 @@
+/*
+EmptyBlock
+option = text
```

Comment:
Example using statement.

Reply:
We have this already (The input file that I modifed in this PR)
https://github.com/checkstyle/checkstyle/blob/3369a7ab5dd8e07d5299e6016a88e38e0cba09a9/src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/blocks/emptyblock/InputEmptyBlockSwitchExpressions.java